cuda/test/base/CMakeLists.txt

```
@@ -1,13 +1,13 @@
 ginkgo_create_cuda_test(array)
 ginkgo_create_cuda_test(cuda_executor)
-ginkgo_create_test(index_set)
+ginkgo_create_cuda_test(index_set)
 if(GINKGO_HAVE_HWLOC)
     find_package(NUMA REQUIRED)
     ginkgo_create_cuda_test(cuda_executor_topology ADDITIONAL_LIBRARIES NUMA::NUMA)
-endif()
+endif ()
```

```suggestion
endif()
```
because other modifications in this pr are from `endif ()` to `endif()`